client/routes.go

```diff
@@ -1,13 +0,0 @@
-package client
```

Comment:
This change is not mentioned in the changelog yet it should be

Reply:
Sure, will add this to changelog